standardize_mutation_data.py

```diff
@@ -1152,24 +1159,24 @@ def resolve_vcf_allele_depth_values(mapped_sample_format_data, vcf_alleles, vari
     vcf_dp_value = process_datum(mapped_sample_format_data.get("DP", ""))
     # Sanity check that REF/ALT allele depths are lower than the total depth
     if (
-        "DP" in vcf_format_data_keys and vcf_dp_value != ""
+        "DP" in vcf_format_data_keys and not is_missing_vcf_data_value(vcf_dp_value)
         and (
-                ref_count != "" and int(ref_count) > int(vcf_dp_value)
-                or (alt_count != "" and int(alt_count) > int(vcf_dp_value))
-                or (ref_count != "" and alt_count != "" and ((int(ref_count) + int(alt_count)) > int(vcf_dp_value)))
+                not is_missing_vcf_data_value(ref_count) and float(ref_count) > float(vcf_dp_value)
+                or (not is_missing_vcf_data_value(alt_count) and float(alt_count) > float(vcf_dp_value))
+                or (not is_missing_vcf_data_value(ref_count) and not is_missing_vcf_data_value(alt_count) and ((float(ref_count) + float(alt_count)) > float(vcf_dp_value)))
             )
         ):
-        mapped_sample_format_data["DP"] = str(sum(map(int, allele_depth_values)))
+        mapped_sample_format_data["DP"] = str(sum(map(float, allele_depth_values)))
 
     # if we have REF/ALT allele depths, but no DP, then set DP equal to the sum of all ADs
     if (
-        (ref_count != "" and alt_count != "")
+        (not ref_count in NULL_OR_MISSING_VCF_VALUES and not is_missing_vcf_data_value(alt_count))
```

Comment:
change the first term to : `not is_missing_vcf_data_value(ref_count)`

Reply:
updated